docs/source/guide/prompts_model.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Create a model
+short: Create a model
+tier: enterprise
+type: guide
+order: 0
+order_enterprise: 228
+meta_title: Create a model
+meta_description: How to create a Prompts model
+section: Prompts
+date: 2024-06-11 16:53:16
+---
+
+
+## Prerequisites
+
+* An OpenAI API key. 
+* You need to have a project that meets the following criteria:
+  * Text-based data set (meaning you are annotating text and not image or video files). 
+  * The labeling configuration for the project must be set up to use single-class classification (`choice="single"`). 
+  * You should have at least one task with a [ground truth annotation](quality#Define-ground-truth-annotations-for-a-project). Only tasks with a ground truth will be listed in the prompt interface. 
```

Comment:
maybe this can be conditional - like "if you want to evaluate for accuracy for your prompt, you'll need GT"